### ISSUE TITLE
Requests to one of configured domains get stuck at 'Submitting request'

### ISSUE THREAD

**Issue by ivoo13** [lithnet/access-manager#208]
Hello,

We've got two domains configured for Just-In-Time access. Configuration was working fine for a few months, but recently requests for servers in the second domain are getting stuck with 'Submitting request...' prompt. There is no errors/timeouts in browser or in access-manager-webapp.log. At the same time requests to the first domain (in which Access Manager is hosted) work fine.

Both domains' statuses in Access Manager seem fine:
![obraz](https://github.com/lithnet/access-manager/assets/28542216/c272759a-11df-41ee-858b-2208bcc47669)
![obraz](https://github.com/lithnet/access-manager/assets/28542216/7ac5be4e-e0c0-4c2d-9d58-4e746e5f5e8a)

Each of those problematic requests seems to end on 'Attempting to create S4U AuthorizationContext against server' and there are no entries after that part (nothing about any kind of success/failure/response etc.). See in the attached log:
[Lithnet-issue-anonymized.txt](https://github.com/lithnet/access-manager/files/11935171/Lithnet-issue-anonymized.txt)

The only recent configuration change I am aware of was configuring three roles in Authorization Rules, but they also worked as expected during our tests for a few days before the problem first appeared. Now, if I sign in as user from the problematic domain and click the 'Roles' tab, the page doesn't load (Similarly to picking server from the second domain, logs end on  'Attempting to create S4U AuthorizationContext against server' entry).

We tried to find any clues about the issue on domain controllers, but to no avail. Our network team also verified that there is no traffic blocked between the access manager server and domain controllers.

We are currently running Access Manager version 2.0.9422.0 (updated shortly after issue appeared). 

Do you have any suggestions on what might have caused such issue or what else we can do to diagnose it?

**Comment by ivoo13** [comment]
Thank you. That's probably it. Our Access Manager is running on Windows server 2022 and indeed it had KB5026370 installed some time before we noticed the problem for the first time. (It's odd that MS didn't add info about that issue on https://support.microsoft.com/en-us/topic/may-9-2023-kb5026370-os-build-20348-1726-8c5dc605-d613-46ea-9232-1425cfc91d62, but I probably wouldn't realize it's the cause anyway.)

That also clarifies why we've been seeing increased CPU utilization on that server. I suspected that it might be related, but I didn't find any proof for that before.

I don't think our client will agree to remove security updates, so we will probably just stick to AD group-based access to servers in the affected domain for now. Hopefully MS fixes it this month.
